etn-ccis/blui-react-workflows
# Fix i18n options

Fixes 4591 .



#### Changes proposed in this Pull Request:

- Add Querystring support for language detection 
- Add Localstorage support for language update 
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:
First scenari0
-  Go to http://localhost:3000/login?lng=zh
- This should update the local storage key `app-i18nextLng`(app local storage)  and `blui-i18nextLng` (auth workflow context)
Second scenario
- Click on debug button change the language it should update both local storage key mentioned above.
- Login and got homepage and change  language  this should update language of complete app including login and registration workflow


#### Any specific feedback you are looking for?

-
